### PR TITLE
Update Functions to SwiftUI.

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

File: functions/FunctionsExample/FunctionsExample/View/AddNumbers.swift
Comment (granluo) [reply functions/FunctionsExample/FunctionsExample/View/AddNumbers.swift]: Done, and updated to 2022.

File: functions/FunctionsExample/FunctionsExample/View/AddMessage.swift
Comment (morganchen12) [inline functions/FunctionsExample/FunctionsExample/View/AddMessage.swift:49]: These tag comments can be removed. (here and throughout)

**Review by morganchen12 — COMMENTED** [review]
Run clang format to fix the CI breakage.

File: functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift
Comment (ncooke3) [inline functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift:48]: What do things look like without this manual frame modifier?

File: functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift
Comment (ncooke3) [inline functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift:67]: I think we should be able to achieve the current layout without the manual frame specification by relying solely on SwiftUI's layout engine. I added some comments above to experiment with how we can get there.  cc: @ryanwilson

File: functions/FunctionsExample/FunctionsExample/View/AddNumbers.swift
Comment (ncooke3) [inline functions/FunctionsExample/FunctionsExample/View/AddNumbers.swift:40]: There is an sf symbol called "plus" that may look better instead of a textual plus.

File: functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift
Comment (ncooke3) [inline functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift:38]: Try using the .secondaryLabelColor color for the description.

File: functions/FunctionsExample/FunctionsExample/View/AddMessage.swift
Comment (ncooke3) [reply functions/FunctionsExample/FunctionsExample/View/AddMessage.swift:20]: Also, do you think we should use  "CapitalizeMessage"  instead of "AddMessage" ?

**Comment by ncooke3** [comment]
Great work @granluo! I added some review above. LMK if you have any questions. I also thought of a neat design for the card views. I will share a sketch with you and see what you're thoughts are.

File: functions/FunctionsExample/FunctionsExample/View/AddMessage.swift
Comment (morganchen12) [reply functions/FunctionsExample/FunctionsExample/View/AddMessage.swift:45]: Yeah, let's do that. If we're modernizing we might as well make everything as modern as possible.

File: functions/FunctionsExample/FunctionsExample/View/AddNumbers.swift
Comment (morganchen12) [inline functions/FunctionsExample/FunctionsExample/View/AddNumbers.swift:75]: nit: Instead of always printing the response here, put it in the else of the if let statement below so it only prints on unexpected behavior.

**Review by morganchen12 — COMMENTED** [review]
LGTM, but please resolve all comments before merging.

File: functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift
Comment (granluo) [reply functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift:67]: Yes, after updating to `padding()`, the manual frame specification can be removed.

File: functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift
Comment (granluo) [reply functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift:48]: It looks like all frame could be removed. We can control the frame through padding.

File: functions/FunctionsExample/FunctionsExample/View/AddMessage.swift
Comment (granluo) [reply functions/FunctionsExample/FunctionsExample/View/AddMessage.swift]: Yep it looks not much different. It looks narrower than the `.frame`, but I can leave `.padding` there and update it when necessary.

File: functions/README.md
Comment (paulb777) [inline functions/README.md:20]: To align with the Quickstart, the `index.js` should be updated to

**Review by paulb777 — APPROVED** [review]
LGTM - Please get at least one more approval before merging.

File: functions/README.md
Comment (morganchen12) [reply functions/README.md:20]: also, `quickstart` should be lowercase (here and throughout).

File: functions/README.md
Comment (morganchen12) [inline functions/README.md:44]: `run` -> `ran`

**Review by morganchen12 — APPROVED** [review]

**Comment by granluo** [comment]
> LGTM - Please get at least one more approval before merging.

Will merge tomorrow to avoid failures of nightly testings.

File: functions/FunctionsExample/FunctionsExample/View/CapitalizeMessageView.swift
Comment (ncooke3) [inline functions/FunctionsExample/FunctionsExample/View/CapitalizeMessageView.swift:47]: Should we use `FirebaseFunctionsSwift` to showcase the recent Codable-related changes? This code could be further simplified like:
https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseFunctionsSwift/Tests/IntegrationTests.swift#L156-L163

Imo, I think we should use the `FirebaseFunctionsSwift` extension, mainly to showcase Swift API improvements as they are released. cc: @morganchen12

File: functions/README.md
Comment (ncooke3) [inline functions/README.md:21]: optional: add "js" to code block for js syntax highlighting
```suggestion
```js
```

**Review by ncooke3 — APPROVED** [review]
LGTM!

File: functions/FunctionsExample/FunctionsExample/View/CapitalizeMessageView.swift
Comment (granluo) [reply functions/FunctionsExample/FunctionsExample/View/CapitalizeMessageView.swift]: Good point, just add it and the new function is neat.

**Pull request merged**
